### ISSUE TITLE
Refresh button in table view

### ISSUE THREAD

**Issue by zycon** [lensapp/lens#245]
**What would you like to be added**: It would be nice if there's an refresh button to reload the table view shown 

**Why is this needed**: Sometimes i see the view is not properly reflected , and after restarting of lens , it works

**Environment you are Lens application on:**

- Kubernetes distribution: [EKS, AKS, GKE, Rancher, ...] GKE
- Desktop OS: <the OS Lens app runs on in your case> MAC OS

**Comment by jakolehm** [comment]
Workaround: `View -> Reload` (or related keyboard shortcut).

**Comment by jakolehm** [comment]
@zycon are you still missing refresh button?

**Comment by tg44** [comment]
I found this issue bcs I missing this button... The workaround with view->refresh refreshes everything (like the terminal too). Clicking to another submenu and back usually solves the problem, but it would be easier to just hit a button. (Also as an alternative it would be ok for me at least if the currently selected menu would be still clickable, and would reload the table.)

**Comment by tg44** [comment]
Also, as I found out it not just the tales. The service endpoints sometimes stay in a stable state (and not refreshing after closing the sidebar and click it again), where tools like the kubernetes dashoard can be refreshed more simply. Overall I like this UI, but I still need to use both tools to get the most recent picture :(

**Comment by Nokel81** [comment]
This shouldn't be necessary, sounds like a duplicate of https://github.com/lensapp/lens/issues/3821 and maybe also of https://github.com/lensapp/lens/issues/4897.

**Comment by tg44** [comment]
I agree that it shouldn't be necessary but probably would be easy to implement and would solve a lot of fustration overall. Ppl like clicking the refresh button.
